.github/workflows/quay.yml

```diff
@@ -29,6 +29,7 @@ on:
     branches:
       - master
       - stage_*
+      - v*
```

Comment:
Please comment, I'm not sure I understand correctly.

Reply:
I removed this as tghis was a test of GitHub actions.